deployments/sandbox/environment.py

```diff
@@ -96,8 +96,6 @@ def env() -> Mapping[str, Optional[str]]:
             }
         ),
 
-        'AZUL_DSS_QUERY_PREFIX': '42',
```

Comment:
What does this have to do with #2843, the issue referenced in the title of the commit that this change will be squashed into? AFAICT, this is a left-over from an already merged PR, so this change should be in a commit whose title references the issue that other PR addressed.